### ISSUE TITLE
Hover over bars in threshold filtering and see those values (genetic distances)

### ISSUE THREAD

**Issue by billswitzer2** [CDCgov/MicrobeTrace#392]
As a User I want to be able to hover of the bars in the filtering threshold so I  can see those distances and know which ones to select more easily.

##**Acceptance Criteria**

Given a fasta file of sequencers, a newick file, or distance matrix and go to the filtering threshold in global settings, I want to hover my pointer over a bar in the histogram and see what that distance is without clicking on the bar.

This would require moving the pop-up message that shows what the filtering threshold is for.

**Comment by roxycintron** [comment]
See issue https://github.com/CDCgov/MicrobeTrace/issues/617

**Comment by ikb6** [comment]
@roxycintron @mossy426 , this feature works for me with both TN93 as well as SNP edgelists.  I am able to hover over histogram and change thresholds successfully. This is in PROD- just tried it.

**Comment by roxycintron** [comment]
In DEV/PROD, I used Anu's node list with aligned sequences and selected TN93 and it doesn't work. I also used my hantavirus distance edge file and fasta file which have a smaller dataset with a distance range of 0-0.06. In this case, TN93 doesn't work. It seems to work with larger hantavirus datasets but it isn't consistent.

I confirmed it's working with Anu's SNP edge list.

Will need to test all file types again after bug fix.
